source/clear-linux/tutorials/kata.rst

```diff
@@ -28,24 +28,23 @@ Install Kata Containers
 ***********************
 
 Kata Containers is included in the :file:`containers-virt` bundle. To install the
-framework, enter:
+framework, enter the following command:
 
 .. code-block:: bash
 
    sudo swupd bundle-add containers-virt
 
-Configure Docker\* to use Kata Containers by default
+Configure Docker\* to use Kata Containers by default.
 
 .. code-block:: bash
 
    sudo mkdir -p /etc/systemd/system/docker.service.d/
-   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/51-runtime.conf
    [Service]
-   ExecStart=
-   ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+   Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime kata-runtime"
```

Comment:
JFYI, configuring `/etc/docker/daemon.json` is another option to enable Kata Container as the default runtime 
https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file

```json
{
    "default-runtime": "kata-runtime",
    "runtimes": {
        "kata-runtime": {
            "path": "/usr/bin/kata-runtime"
        }
    }
}
```

Reply:
@MCamp859: The kata_migration tutorial is intended for users that are already using Clear Containers. These are planned to be deprecated in the following weeks so this tutorial should be also (or at least be archived). Only the kata tutorial will be kept alive in the future, so no need to merge them.

Reply:
@devimc 
Thanks for presenting an alternative. I need clarity on your recommendations, please as to whether you recommend this alternative be added, or not. 

@joselamego 
We need to provide the best option for our users to migrate. Can you please provide your opinion if Mary's contribution is acceptable? It's not at all clear to me that from this previous dialogue both you and devimc agreed. I just need confirmation. Thank you.  

I agree that we don't need to give alternatives, given the potentially short life-span of this document. Nevertheless, we should provide current CL users with the simplest and best instructions on how to migrate.

Reply:
@mvincerx 
My comment was related to the suggestion of combining both tutorials, which I believe is not necessary. Regarding the alternative to configure the runtime, I suggest to add it as a note, leaving the current instruction as default.

Reply:
Excellent. Thanks for clarifying.  @MCamp859 can you please follow up and add this as a note?

Reply:
What's the alternative? The .json file? I don't think that's necessary.